charts/rancher-eks-operator/104.0.1+up1.9.0/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+annotations:
+  catalog.cattle.io/auto-install: rancher-eks-operator-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>= 1.23.0-0 < 1.31.0-0'
+  catalog.cattle.io/namespace: cattle-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux,windows
+  catalog.cattle.io/provides-gvr: eksclusterconfigs.eks.cattle.io/v1
+  catalog.cattle.io/rancher-version: '>= 2.8.0-0 < 2.10.0-0'
```

Comment:
```suggestion
  catalog.cattle.io/rancher-version: '>= 2.9.0-0 < 2.10.0-0'
```

Reply:
weird aks and gke had this already updated

Reply:
good catch, strange that only eks  was affected